_posts/2019-10-30-cockpit-206.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Cockpit 206
+author: Gundersanne
+date: 2019-10-30
+tags: cockpit linux
+slug: cockpit-206
+category: release
+summary: Cockpit with machines, firewall, and login page improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.
+
+### Machines: NIC deletion
+
+NICs can now be removed from virtual machines.
+
+![Machines NIC deletion](/images/machines-nic-deletion.png)
+
+### login: Check reuse password by default
+
+Cockpit is heavily restricted without sudo rights, so make reusing the password
```

Comment:
It can also be policykit, so perhaps s/sudo/administrative or root/ here? Also, the point is less that "it is restricted", but that its main use case is to troubleshoot and change the system state, thus most of which doesn't work without root privileges.